src/components/CoursesItem/CoursesItem.tsx

```diff
@@ -0,0 +1,92 @@
+import Hls from 'hls.js';
+import { useEffect } from 'react';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import { onHoverElement } from '../../helpers/hoverHelper';
+import { ICoursesItemComponentProps } from '../../interfaces/CoursesItem.interfaces';
+import {
+  ListItemS,
+  LinkItemS,
+  ImageContainerS,
+  TitleS,
+  TextS,
+  SkillsListS,
+  SkillTitleS,
+  SkillsItemS,
+} from './CoursesItem.styled';
+
+const CoursesItem = ({ course }: ICoursesItemComponentProps) => {
+  const { id, previewImageLink, meta, title, lessonsCount, rating } = course;
```

Comment:
```suggestion
  const { 
    id, 
    previewImageLink, 
    meta, 
    title, 
    lessonsCount, 
    rating,
  } = course;
```
На мою думку, таке форматування виглядає краще при великій кількості пропсів

Reply:
это задача линтеров